cre.cpp

```diff
@@ -2724,12 +2721,10 @@ static int getPageLinks(lua_State *L) {
 	lua_newtable(L); // all links (actual entries may be less than links.length(), so, no pre-alloc)
 
 	ldomXRangeList links;
-	ldomXRangeList & sel = doc->text_view->getDocument()->getSelections();
```

Comment:
(I was a bit worried these selections() methods may do/have wanted side effects when called, but this one just returns a reference.)
Anyway, keep this line commented out, as below we have (which may be useful):
https://github.com/koreader/koreader-base/blob/adafa0bbf8c39b5f0455c3a2754a924a0b5556fb/cre.cpp#L2801-L2805
and may be line 2732 too (it may be needed to clear past ones if we _need to visually see new ones when debugging_).